src/SignalR/common/Shared/MessageBuffer.cs

```diff
@@ -19,6 +19,8 @@ internal sealed class MessageBuffer : IDisposable
 {
     private static readonly TaskCompletionSource<FlushResult> _completedTCS = new TaskCompletionSource<FlushResult>();
 
+    public static readonly TimeSpan AckRate = TimeSpan.FromSeconds(1);
```

Comment:
```suggestion
    public static TimeSpan AckRate => TimeSpan.FromSeconds(1);
```
Perf-/codegen-wise the property is better here, as the JIT is able to [fold this into a constant](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIGYACMhgYQYG8aHunHjykDACoBLfDADKAB2wA7BgEEwAawBK2DDAYBeAHzCxkmbIB0AMSgR8EmJFkATXAApyASgDcNAL5A), whilst for the static readonly a memory access is needed.

This may not be relevant here, as that `TimeSpan` is only used in the ctor, so treat it as just FYI...